broker/src/main/java/io/zeebe/broker/system/configuration/RocksdbCfg.java

```diff
@@ -64,8 +65,17 @@ public void setMemoryLimit(final DataSize memoryLimit) {
     this.memoryLimit = memoryLimit;
   }
 
+  public int getMaxOpenFiles() {
+    return maxOpenFiles;
+  }
+
+  public void setMaxOpenFiles(final int maxOpenFiles) {
+    this.maxOpenFiles = maxOpenFiles;
+  }
+
   public RocksDbConfiguration createRocksDbConfiguration() {
-    return RocksDbConfiguration.of(columnFamilyOptions, statisticsEnabled, memoryLimit.toBytes());
+    return RocksDbConfiguration.of(
```

Comment:
I can see that this won't scale eventually :sweat_smile: 

At what point do we want to make it a builder or even just a mutable config object that can be chained, e.g.

```java
public final class RocksDbConfiguration {
  // ... stuff

  public RocksDbConfiguration withStatisticsEnabled(final boolean enabled) {
     // ...
  }

  // ... more stuff
}
```

This is more of a question than a change request or anything

Reply:
Since I'm done not now :laughing: 5+ I can imagine :)